.github/workflows/test.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10, 12, 14]
+        node-version: [20, 22]
```

Node.js 18 is still an active LTS version, so we should probably add it as well.
```suggestion
        node-version: [18, 20, 22]
```

---

I think the builtin test reporter doesn't work on v18. But I cannot recall, so I'll need to check on this.

---

On v18, lcov doesn't seem to be there:
```
> node --test --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info

node:internal/test_runner/harness:41
      throw err;
      ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'lcov' imported from /Users/rolandgroza/Work/attr-accept/
    at new NodeError (node:internal/errors:405:5)
    at packageResolve (node:internal/modules/esm/resolve:916:9)
    at moduleResolve (node:internal/modules/esm/resolve:973:20)
    at defaultResolve (node:internal/modules/esm/resolve:1206:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:404:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:373:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:250:38)
    at ModuleLoader.import (node:internal/modules/esm/loader:336:34)
    at node:internal/test_runner/utils:154:40
    at node:internal/per_context/primordials:523:39 {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Not sure if it can be loaded some other way.